pcweb/components_webpage/footer.py

```diff
@@ -0,0 +1,195 @@
+import reflex as rx
+from pcweb import constants, styles
+from pcweb.components.logo import logo
+from pcweb.pages.docs import getting_started
+from pcweb.pages.docs.gallery import gallery
+from pcweb.pages.docs.library import library
+from pcweb.pages.index import index
+
+footer_item_style = {
+    "font_family": styles.SANS,
+    "font_weight": "500",
+    "_hover": {"color": "#82799E"},
+    "color": "#6C6C81"
+}
+
+footer_style = {
+    "box_shadow": "medium-lg",
+    "border_top": "0.1em solid #82799E",
+    "vertical_align": "bottom",
+    "padding_top": "4em",
+    "padding_bottom": "2em",
+    "padding_x": styles.PADDING_X2,
+    "bg": "#110F1F",
+}
+
+def prompt_sign():
+    return rx.chakra.text(
+        "$",
+        color=styles.ACCENT_COLOR,
+        font_family=styles.SANS,
+        style={"userSelect": "none"},
+    )
+
+def container(*children, **kwargs):
+    kwargs = {"max_width": "1440px", "padding_x": ["1em", "2em", "3em"], **kwargs}
+    return rx.chakra.container(
+        *children,
+        **kwargs,
+    )
+
+
+number_color = "#4E5A6A"
+text_color="#A9ABD8"
+other_color ="#5646ED"
+bottom_color = "#6C6C81"
+
+def installation():
```

Comment:
Same with this, isn't it the same as the other footer?

Reply:
Deleted other footer file extra from my refactor